ch.protonmail.protonmail-bridge.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: ch.protonmail.protonmail-bridge
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "18.08"
+command: protonmail-bridge
+rename-desktop-file: ProtonMail_Bridge.desktop
+rename-icon: ProtonMail_Bridge
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
Does this actually have notifications?

Reply:
I haven't used it, but there is a notification setting for outgoing mails without encryption. I think it also gives a notification when there is a update.